src/Gui/ApplicationPy.cpp

```diff
@@ -1267,39 +1267,20 @@ PyObject* Application::sGetMarkerIndex(PyObject * /*self*/, PyObject *args)
     PY_TRY {
         ParameterGrp::handle const hGrp = App::GetApplication().GetParameterGroupByPath("User parameter:BaseApp/Preferences/View");
 
-        //find the appropriate marker style string token
-        std::string marker_arg = pstr;
-
-        std::list<std::pair<std::string, std::string> > markerList = {
-            {"square", "DIAMOND_FILLED"},
-            {"cross", "CROSS"},
-            {"plus", "PLUS"},
-            {"empty", "SQUARE_LINE"},
-            {"quad", "SQUARE_FILLED"},
-            {"circle", "CIRCLE_LINE"},
-            {"default", "CIRCLE_FILLED"}
-        };
-
-        std::list<std::pair<std::string, std::string>>::iterator markerStyle;
-
-        for (markerStyle = markerList.begin(); markerStyle != markerList.end(); ++markerStyle)
-        {
-            if (marker_arg == (*markerStyle).first || marker_arg == (*markerStyle).second)
-                break;
-        }
-
-        marker_arg = "CIRCLE_FILLED";
-
-        if (markerStyle != markerList.end())
-            marker_arg = (*markerStyle).second;
-
-        //get the marker size
-        int sizeList[]={5, 7, 9};
-
-        if (std::find(std::begin(sizeList), std::end(sizeList), defSize) == std::end(sizeList))
-            defSize = 9;
-
-        return Py_BuildValue("i", Gui::Inventor::MarkerBitmaps::getMarkerIndex(marker_arg, defSize));

```

Comment:
The changes here look like a merge/rebase issue - wanna check out the latest version from `master` and replace this one?

Reply:
@mlampert 
>The changes here look like a merge/rebase issue - wanna check out the latest version from master and replace this one?

Sir, I have never changed: 
*src/Mod/AddonManager/AddonManager.py
*src/Mod/Sketcher/Gui/CommandSketcherTools.cpp 
*src/Mod/Sketcher/Gui/CommandSketcherTools.cpp 
*src/Gui/ApplicationPy.cpp 

If I understand you correctly, you would like me to copy each of the files mentioned FROM current `master` to my fork `master` so these issues will be resolved?